device/dhcp/clients/v4/dhcpv4clientadvanced.yaml

```diff
@@ -0,0 +1,58 @@
+components:
+  schemas:
+    Device.Dhcpv4Client.Advanced:
```

Comment:
This looks identical to previous parameters ... is this used anywhere at all ?

Reply:
no will delete it.